LOGGING/src/ILogger/ApplicationInsightsLoggerEventSource.cs

```diff
@@ -0,0 +1,43 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="ApplicationInsightsLoggerEventSource.cs" company="Microsoft">
+// Copyright (c) Microsoft Corporation. 
+// All rights reserved.  2013
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Microsoft.Extensions.Logging.ApplicationInsights
+{
+    using System.Diagnostics.Tracing;
+    using System.Reflection;
+
+    /// <summary>
+    /// EventSource for reporting errors and warnings from Logging module.
+    /// </summary>
+    [EventSource(Name = "Microsoft-ApplicationInsights-Logger-EventSourceListener")]
```

Comment:
EventSourceListener is a typo?
I'd call it Microsoft-ApplicationInsightsLoggerProvider or something like that

Reply:
https://github.com/microsoft/ApplicationInsights-Home/tree/master/Samples/ETW can you update it here as well? 
And docs too: https://docs.microsoft.com/en-us/azure/azure-monitor/app/asp-net-troubleshoot-no-data#PerfView

Reply:
Yes, it's  a typo, tried changing from constant to normal on Event source decoration and added listener additionally. 

Good call out to update other articles, was thinking about it. Not sure, Home repo will allow an update as it is in archived state.